google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -336,6 +336,9 @@ clouddebug.runconfig.formoredetails=For more details,
 clouddebug.runconfig.documentation.url=https://cloud.google.com/debugger/
 clouddebug.runconfig.documentation.url.text=click here
 
+gcs.panel.toolbar.refresh.hover.text=Refresh
```

Comment:
I like this. But how to do feel about adding this as an enhancement after the initial implementation? I wont block the PR on this unless you feel strongly about doing this now.